TLM/TLM/Patch/_PathManager/WaitForAllPathsPatch.cs

```diff
@@ -0,0 +1,19 @@
+namespace TrafficManager.Patch._PathManager {
+    using Custom.PathFinding;
+    using HarmonyLib;
+    using JetBrains.Annotations;
+
+    [HarmonyPatch(typeof(PathManager), nameof(PathManager.WaitForAllPaths))]
+    public class WaitForAllPathsPatch {
+
+        [UsedImplicitly]
+        public static bool Prefix() {
+            if (CustomPathManager._instance) {
```

Comment:
am I oldschool for wanting a != null / is not null check? ;D

Reply:
`_instance` is `MonoBehaviour`, and my glorious IDE reported expensive null check which... doesn't really matter since that method will be called once or twice, assuming no hot reload or reloading savegame. 
Will change that 😉